### PR TITLE
Run tests on php 7.4

### PR DESCRIPTION
**Pull request by ihor-sviziev**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4
 install: composer install --no-interaction --prefer-source
```
Comment (ihor-sviziev) [inline .travis.yml:9]: Btw why we’re  using —prefer-source and not —prefer-dist?

Comment (lenaorobei) [reply .travis.yml:9]: I believe because we are doing git-based installation. Is there a benefit from using `prefer-dist` option in our context?

Comment (ihor-sviziev) [reply .travis.yml:9]: Yes, installing from source mean that composer will clone all packages as a fit repo, while “dist” means that packages will be tried to be downloaded as zip files exact commit and just unzipped, usually it much faster

Comment (ihor-sviziev) [reply .travis.yml:9]: I just created PR with changing to dust, let’s see if it will work

**Pull request merged**